SoftLayer/CLI/routes.py

```diff
@@ -119,6 +119,9 @@
     ('block:volume-convert', 'SoftLayer.CLI.block.convert:cli'),
     ('block:volume-set-note', 'SoftLayer.CLI.block.set_note:cli'),
 
+    ('email', 'SoftLayer.CLI.email'),
+    ('email:detail', 'SoftLayer.CLI.email.detail:cli'),
```

Comment:
I think the command should be called `email:list` since it lists all the accounts, and not one specific account.